tests/test_01_create_mock_data.py

```diff
@@ -0,0 +1,21 @@
+"""
+Tests for the create_mock_data module.
+"""
+
+import xarray as xr
+
+from dmd_era5.create_mock_data import create_mock_era5
+
+
+def test_create_mock_era5():
+    """Test that the create_mock_era5 function correctly creates a mock ERA5 dataset."""
+    mock_ds = create_mock_era5(
+        start_datetime="2019-01-01",
+        end_datetime="2019-01-02",
+        variables=["temperature"],
+        levels=[1000, 850, 500],
+    )
+
+    assert isinstance(mock_ds, xr.Dataset)
+    assert "temperature" in mock_ds.data_vars
+    assert list(mock_ds.level.values) == [1000, 850, 500]
```

Comment:
Recommend printing out some error message if it makes sense, although perhaps not necessary since this is under the test module... e.g.,

assert (
    isinstance(mock_ds, xr.Dataset),
    f"moc_ds is not of an XR.Dataset instance, but of type {type(mock_ds)}"
)

Reply:
Apologies, I should have started a review instead of adding as a single comment!

Reply:
Yes, that's good practice and it's what we have been doing in most (although apparently not all) `assert` statements

Reply:
Done